specification/elastic/resource-manager/Microsoft.Elastic/preview/2023-07-01-preview/elastic.json

```diff
@@ -0,0 +1,2763 @@
+        },
+        "systemData": {
+          "readOnly": true,
+          "description": "The system metadata relating to this resource",
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
+        }
+      }
+    },
+    "MonitoringTagRulesListResponse": {
+      "description": "Response of a list operation.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Results of a list operation.",
+          "type": "array",
+          "x-ms-identifiers": [],
+          "items": {
+            "$ref": "#/definitions/MonitoringTagRules"
+          }
+        },
+        "nextLink": {
+          "description": "Link to the next set of results, if any.",
+          "type": "string"
+        }
+      }
+    },
+    "BillingInfoResponse": {
+      "description": "Marketplace Subscription and Organization details to which resource gets billed into.",
+      "type": "object",
+      "properties": {
+        "marketplaceSaasInfo": {
+          "description": "Marketplace Subscription details",
+          "$ref": "#/definitions/MarketplaceSaaSInfo"
+        },
+        "partnerBillingEntity": {
+          "description": "Partner Billing Entity details: Organization Info",
+          "$ref": "#/definitions/PartnerBillingEntity"
+        }
+      }
+    },
+    "MarketplaceSaaSInfo": {
+      "description": "Marketplace SAAS Info of the resource.",
+      "type": "object",
+      "properties": {
+        "marketplaceSubscription": {
+          "description": "Marketplace Subscription",
+          "type": "object",
+          "properties": {
+            "id": {
+              "description": "Marketplace Subscription Id. This is a GUID-formatted string.",
+              "type": "string"
+            }
+          }
+        },
+        "marketplaceName": {
+          "description": "Marketplace Subscription Details: SAAS Name",
+          "type": "string"
+        },
+        "marketplaceResourceId": {
+          "description": "Marketplace Subscription Details: Resource URI",
+          "type": "string"
+        },
+        "marketplaceStatus": {
+          "description": "Marketplace Subscription Details: SaaS Subscription Status",
+          "type": "string"
+        },
+        "billedAzureSubscriptionId": {
+          "description": "The Azure Subscription ID to which the Marketplace Subscription belongs and gets billed into.",
+          "type": "string"
+        },
+        "subscribed": {
+          "description": "Flag specifying if the Marketplace status is subscribed or not.",
+          "type": "boolean"
+        }
+      }
+    },
+    "PartnerBillingEntity": {
+      "description": "Partner Billing details associated with the resource.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The Elastic Organization Id.",
+          "type": "string"
+        },
+        "name": {
```

Comment:
>name [](http://example.com/codeflow?start=9&length=4)

Similar comment here: recommend something more specific... elasticOrganizationName?

Reply:
This is a common component therefore contract/model is common, and this partner entity naming could be different for different Liftr partners. Therefore, we went for common naming convention therefore won't be able to rename.